packages/transport/src/modules/sync/SynchronizedCollection.ts

```diff
@@ -87,6 +88,23 @@ export class SynchronizedCollection implements IDatabaseCollection {
 
         const newObjectJson = newObject.toJSON();
 
+        if (typeof process === "object" && process.env.CI === "true") {
```

Comment:
```suggestion
        if (typeof globalThis.process === "object" && globalThis.process.env.CI) {
```

two things:
- used globalThis to be clear about where process should be defined or not
- don't exactly pin CI (we don't care what's inside this var)